docs/requirements.txt

```diff
@@ -2,3 +2,4 @@ git+https://github.com/sphinx-doc/sphinx@2.0
 sphinxcontrib-napoleon
 alabaster
 nbsphinx
+pyyaml
```

Comment:
How is this being used?

Reply:
In `horovod/run/run.py` there is `import yaml`
So before building doc we should install this dependency first, otherwise doc building will fail.